file_paths.md

```diff
@@ -161,6 +165,11 @@ create a `puppet` user or group.
     Path                                      Setting                        3.x
     C:\ProgramData                                                           n/a
 
+    C:\ProgramData\PuppetLabs/client-tools    # default client tool settings
```

Comment:
There's a '/' where there should be a '\'

Do we need a config dir here? like the "C:\ProgramData\PuppetLabs\puppet\etc" below? I'm not sure what else might land in the C:\ProgramData\PuppetLabs\client-tools directory